src/contracts/FoxStaking.sol

```diff
@@ -206,11 +207,20 @@ interface IWarmup {
     function retrieve(address staker_, uint256 amount_) external;
 }
 
+interface ITokePool {
+    function deposit(uint256 amount) external;
+
+    function balanceOf(address owner) external view returns (uint256);
+}
+
 contract FoxStaking is Ownable {
     using SafeMath for uint256;
     using SafeERC20 for IERC20;
 
-    address public immutable FOX;
+    // TODO: what if tFOX pool address is updated, we should allow this to be updated as well
+    address public constant tokePool =
+        0x808D3E6b23516967ceAE4f17a5F9038383ED5311;
+    address public immutable FOX = 0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d;
```

Comment:
see previous  comment re: hardcoding FOX

Reply:
would even recommend that we actually use a variable name different than FOX in the case we do deploy this for other DAOs in the future.

Reply:
https://github.com/shapeshift/foxy/issues/19